cpp/velox/benchmarks/BenchmarkUtils.cc

```diff
@@ -28,19 +28,24 @@ namespace fs = std::filesystem;
 
 DEFINE_bool(print_result, true, "Print result for execution");
 DEFINE_string(write_file, "", "Write the output to parquet file, file absolute path");
+DEFINE_string(batch_size, "4096", "To set velox::core::QueryConfig::kPreferredOutputBatchSize.");
```

Comment:
Do we also want to update batch size for [WholeStageResultIterator](https://github.com/oap-project/gluten/blob/0af5406edf7431e902532660ef46f662fafbe4f6/cpp/velox/compute/WholeStageResultIterator.cc#L198) besides benchmark?

Reply:
Updated.